easybuild/easyblocks/o/openssl.py

```diff
@@ -80,14 +80,14 @@ def __init__(self, *args, **kwargs):
             openssldir_cmd = "openssl version -d"
 
             try:
-                (out, _) = run_cmd(openssldir_cmd, log_all=True, simple=False, trace=False)
-                openssldir = openssldir_regex.search(out).group(1)
+                res = run_shell_cmd(openssldir_cmd, hidden=True)
+                openssldir = openssldir_regex.search(res.output).group(1)
             except EasyBuildError:
```

Comment:
`run_shell_cmd` raises an `RunShellCmdError` if the command failed, not `EasyBuildError` like `run_cmd` did

That's probably something we should clarify in the documentation at https://docs.easybuild.io/easybuild-v5/run_shell_cmd too...

Reply:
done in 7edeb37